src/main/scala/mesosphere/marathon/event/http/SubscribersKeeperActor.scala

```diff
@@ -2,43 +2,38 @@ package mesosphere.marathon.event.http
 
 import akka.actor.{ Actor, ActorLogging }
 import akka.pattern.pipe
-import mesosphere.marathon.event.{
-  MarathonSubscriptionEvent,
-  Unsubscribe,
-  Subscribe
-}
 import mesosphere.marathon.event.http.SubscribersKeeperActor._
-import mesosphere.marathon.state.MarathonStore
+import mesosphere.marathon.event.{ MarathonSubscriptionEvent, Subscribe, Unsubscribe }
+import mesosphere.marathon.state.EntityStore
+
 import scala.concurrent.Future
 
-class SubscribersKeeperActor(val store: MarathonStore[EventSubscribers]) extends Actor with ActorLogging {
+class SubscribersKeeperActor(val store: EntityStore[EventSubscribers]) extends Actor with ActorLogging {
```

Comment:
Your changes made me think that MarathonStore can actually be package private. What do you think?